mms/serving_frontend.py

```diff
@@ -182,20 +182,20 @@ def add_endpoint(self, api_definition, callback, **kwargs):
 
     def setup_openapi_endpoints(self, host, port):
         """
-        Firstly, construct Openapi compatible api definition for 
+        Firstly, construct Openapi compatible api definition for
         1. Predict
         2. Ping
         3. API description
-        
+        4. Root (a default for the server, reuses ping functionality)
         Then the api definition is used to setup web server endpoint.
 
         Parameters
         ----------
         host : str
-            Host that server will use 
+            Host that server will use
 
         port: int
-            Host that server will use 
+            Host that server will use
         """
         modelservices = self.service_manager.get_loaded_modelservices()
         # TODO: not hardcode host:port
```

Comment:
Is this issue still relevant, or should we remove the comment?
If still relevant please file an issue and add the link here.